### DISCUSSION TITLE
Request Hero animation, pagination page From REST API

### DISCUSSION THREAD

**Discussion by bobwatcherx** [zauberzeug/nicegui#775] [Ideas / Feature Requests]
Is there a feature like the one I mentioned above. And for page pagination so let's say I request api and get 100 data. And I want to display only 10 data per page. And there is Button to move page. Thanks

**Comment by rodja** [comment]
We do something like that in https://github.com/zauberzeug/nicegui/blob/main/examples/infinite_scroll/main.py. Does that help?
